lab1/Parser.py

```diff
@@ -0,0 +1,49 @@
+from constants.config import config
+from constants.tasks import tasks
+
+
+class Parser:
+    def __init__(self, args):
+        self.args = args
+        self.commands = []
+
+    def get_params(self):
+        if "-u" in self.args:
+            config['TYPE_OF_PROTOCOL'] = '-u'
```

Comment:
nu cred că e o idee bună să modifici o variabilă pe care o imporți din alt fișier (în cazul acesta `config`)

o idee ce-mi vine e să ai o clasă care să inițializeze un config ca și acela pe care-l ai curent, și apoi să modifici info din el în caz de necesitate.

```python
class Parser:
    def __init__(self, args):
        self.config = ConfigurationClass()
        ...
    self get_params(self):
        if '-u' in self.args:
            self.config.change_datagram_type(type='-u')
        ...
```    
or something of the sorts